archivy/config.py

```diff
@@ -1,3 +1,24 @@
+#!/usr/bin/env python
+"""
```

Comment:
Do we need to add this huge chunk to each file?

Reply:
Not a requirement, see https://opensource.stackexchange.com/q/292/

By convention, developers add copyright to each and every file, with the name of the person, if applicable or the name of the project. For example. `The Archivy Project` represents the members of the GitHub organization, i.e you. If some one else have worked on a considerable part of the project and wants to add their copyright to the work, they can append the copy right with
```
(c) 20XX Sponge Bob
```
or something like that. This gives individual contributors finer grained control over their works. If they do not wish to earn a copyright, copyright goes to Archivy Project alone. A developer does not work on all parts of the project, sometimes for example, clemux worked on the tests. Some other contributor cannot claim copyright of the work done by clemux. I hope you get the point.

Mentors have told me to follow this, Sugar Labs, AppImage, Mozilla and Element. So, I guess its a good start. If you feel, it might not be necessary, feel free to rebase / cherry pick the necessary commits.